components/Article/Codebox/index.tsx

```diff
@@ -48,11 +64,34 @@ const Codebox: FC<CodeBoxProps> = ({ children: { props } }) => {
       highlight(codeArray[langIndex], prismLanguage, parsedLanguage)
     );
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [langIndex]);
+  }, [langIndex, codeArray]);
+
+  const ariaLabelText = intl.formatMessage(
```

Comment:
We don't need this `aria-label` for now. screen readers will not use a "copy" button, this button is not "accessible" right now